src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -50,9 +50,9 @@ int find_output(UniValue obj, int n) {
 
 AsyncRPCOperation_sendmany::AsyncRPCOperation_sendmany(
         CMutableTransaction contextualTx,
-        std::string fromAddress,
-        std::vector<SendManyRecipient> tOutputs,
-        std::vector<SendManyRecipient> zOutputs,
+        const std::string& fromAddress,
+        std::vector<SendManyRecipient>&& tOutputs,
+        std::vector<SendManyRecipient>&& zOutputs,
```

Comment:
Thank you, that looks good. But now you need to adjust line 72 accordingly, where you should use the members of the class instead of the parameters that you moved from.

Reply:
Done